tests/bears/GlobalBearTest.py

```diff
@@ -6,9 +6,24 @@
 
 class GlobalBearTest(unittest.TestCase):
 
-    def test_api(self):
-        test_object = GlobalBear(0, Section('name'), None)
-        self.assertRaises(NotImplementedError, test_object.run)
+    def test_file_dict(self):
+        file_dict_0 = {
+            'filename1': 'contents1', 'filename2': 'contents2'
+        }
 
-    def test_kind(self):
+        file_dict_1 = file_dict_0.copy()
+
+        bear = GlobalBear(file_dict_0, Section(''), None)
+
+        self.assertEqual(bear.file_dict, file_dict_0)
+        self.assertEqual(bear.file_dict, file_dict_1)
```

Comment:
@Udayan12167 exactly. @adhikasp I agree that is overkill here, but just to make sure there are no modifications.